roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- block:
+    - copy:
+        src: elasticsearch-logging.yml
+        dest: "{{mktemp.stdout}}/elasticsearch-logging.yml"
+      when: es_logging_contents is undefined
+
+    - copy:
+        src: elasticsearch.yml
+        dest: "{{mktemp.stdout}}/elasticsearch.yml"
+      when: es_config_contents is undefined
+
+    - lineinfile:
+        dest: "{{mktemp.stdout}}/elasticsearch.yml"
+        regexp: '^openshift\.operations\.allow_cluster_reader(.)*$'
+        line: "\nopenshift.operations.allow_cluster_reader: {{openshift_logging_es_ops_allow_cluster_reader | lower}}"
+      when: es_config_contents is undefined
+
+    - copy:
+        content: "{{es_logging_contents}}"
+        dest: "{{mktemp.stdout}}/elasticsearch-logging.yml"
+      when: es_logging_contents is defined
+
+    - copy:
+        content: "{{es_config_contents}}"
+        dest: "{{mktemp.stdout}}/elasticsearch.yml"
+      when: es_config_contents is defined
+
+    - shell: >
+        {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create configmap logging-elasticsearch
+        --from-file=logging.yml={{mktemp.stdout}}/elasticsearch-logging.yml --from-file=elasticsearch.yml={{mktemp.stdout}}/elasticsearch.yml -o yaml --dry-run
+      register: es_configmap
+
+    - copy:
+        content: "{{es_configmap.stdout}}"
+        dest: "{{mktemp.stdout}}/templates/logging-elasticsearch-configmap.yaml"
+      when: es_configmap.stdout is defined
+  check_mode: no
```

Comment:
It looks like all the tasks in this block will always run and display changed, I'm not sure that is the proper behavior here, since it'll report changed tasks even when no changes to the remote system are made. Perhaps these should all be marked `changed_when: no`?